requirements.txt

```diff
@@ -1,2 +1,3 @@
 six>=1.11.0
-ckantoolkit>=0.0.2
\ No newline at end of file
+ckantoolkit>=0.0.2
+itsdangerous==2.0.1
```

Comment:
AFAICT this is Flask related requirement, shouldn't we be using whatever version CKAN core installs?

Reply:
updated